### ISSUE TITLE
GLTF tangents not processed correctly

### ISSUE THREAD

**Issue by CristianSimion5** [nvpro-samples/nvpro_core#52]
The code added in 7688890 seems to be working incorrectly. Specifically, it seems to mess up tangents that have the `w` component equal to `-1.0`. The attached screenshot is from the `vk_raytrace` project with the [NormalTangentMirrorTest](https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/NormalTangentMirrorTest) scene loaded and set to output normals. The correct behaviour should be that all cells have the same normals. I have also tested other scenes and similar problems appear.

![tangent_error](https://github.com/nvpro-samples/nvpro_core/assets/46331290/b482d53a-0f41-4fa1-ac18-817f084afc67)

**Comment by mklefrancois** [comment]
Thank you for reporting the issue, in the added code the `abs` was missing and should have been: 

```
        if(nvmath::nv_sq_norm(nvmath::vec3f(t)) < 0.01F || std::abs(t.w) < 0.5F)
```

This correction will be added shortly.

-mkl

**Comment by mklefrancois** [comment]
Fixed in  8b075d1
